metricflow-semantics/metricflow_semantics/query/query_parser.py

```diff
@@ -81,18 +82,14 @@ class MetricFlowQueryParser:
     TODO: Add fuzzy match results.
     """
 
-    def __init__(  # noqa: D107
-        self,
-        semantic_manifest_lookup: SemanticManifestLookup,
-        where_filter_pattern_factory: WhereFilterPatternFactory = DefaultWhereFilterPatternFactory(),
```

Comment:
Wait, we still use this in MFS. I think all of us want to remove the WhereFilterPatternFactory altogether, but we can't do it just yet unless we move the MFS compatibility shims into MetricFlow itself. In theory we won't need it after end of May so I'm inclined to just wait.

Reply:
Ahh got it! Will update